src/index.js

```diff
@@ -1,41 +1,103 @@
+   const completed = false;
+   const todoList = new todo(description,completed,index);
+   if(description != ''){
+    todosArr.push(todoList);
+    localStorage.setItem('deeds', JSON.stringify(todosArr));
+   }
+}
 
-todos.forEach((deed) => {
-    const content = `
-            <div class="list">
+const Dynamic = document.querySelector('.todo-maker');
+let todos2=() => {
+    const todosArr = storage();
+    Dynamic.innerHTML = '';
+    for (let i = 0; i < todosArr.length; i += 1){
+        let isComplte = todosArr[i].completed == true ? "checked" : "";
+        Dynamic.innerHTML += `
+        <div class="list ${isComplte}">
                 <div class="list-action">
-                    <input type="checkbox">
-                    <p>${deed.description}</p>
+                    <input class ="check" id ="${todosArr[i].index}" ${isComplte} type="checkbox">
+                    <p id ="${todosArr[i].index}" contenteditable="true"class="p">${todosArr[i].description}</p>
                 </div>
                 <div class="action-container">
-                    <div><i class="fa-solid fa-ellipsis-vertical"></i></div>
+                    <div class="icon"><i id = "${todosArr[i].index}" class="fa-solid fa-ellipsis-vertical line"></i> <i id = "${todosArr[i].index}" class="fa-solid fa-trash remove"></i></div>
                 </div>
             </div>
-            <hr>`;
-            Dynamic.innerHTML += content;
-})
\ No newline at end of file
+            `;
+
+    }
+}
+
+todos2();
+
+const checked = document.querySelectorAll('.check');
+checked.forEach((button) => {
+    const todosArr = storage();
+    button.addEventListener('click', (e) => {
+       const card = e.path.filter((el) => el.classList?.contains('list')).at(0);
+       if(button.checked){
+        card.classList.add("checked");
+        todosArr[button.id].completed = true;
+       } else {
+        card.classList.remove("checked")
+        todosArr[button.id].completed = false;
+       }
+       localStorage.setItem('deeds', JSON.stringify(todosArr));
+    });
+  });
+
+document.addEventListener('keyup', (e) => {
+    const todosArr = storage();
+    if (e.key === 'Enter') {
+        localStorage.setItem('deeds', JSON.stringify(todosArr));
+         pushtodo();
+         todos2();
+         window.location.reload();
+        console.log(Dynamic)
+    }
+});
+
+const rmv = document.querySelectorAll('.remove');
+ rmv.forEach((item) => {
+  item.addEventListener('click', (e) => {
+    let todosArr = storage();
+     todosArr.splice(item.id, 1);
+     todosArr.forEach((todo) => {
+        todo.index = todosArr.indexOf(todo);
+     });
+     localStorage.setItem('deeds', JSON.stringify(todosArr));
+     todos2();
+     location.reload();
+      });
+});
+
+// editing tasks
+
+const edit = document.querySelectorAll('.p');
+ edit.forEach((item) => {
+    let todosArr = storage();
+    item.addEventListener('keyup', (e) => {
+    let des = item.innerHTML;
```

Comment:
- [ ] **[OPTIONAL]:** You did a good job working on this milestone 🚀 However, I strongly advise you to use `item.textContent` instead of `item.innerHTML`. This is because when you press `enter` to finish editing a to-do, the innerHTML node of this element registers a `<div></div>` and this is saved to the local storage.

![image](https://user-images.githubusercontent.com/77069011/206558371-aac20d27-91a2-4ab2-bd02-2e6ac06140e4.png)

Please consider fixing this! 🤓 🛠️

Reply:
@anagudelogu all fixed sir, thanks